content/documentation/install-uninstall.md

```diff
@@ -248,133 +257,41 @@ If version number is returned, quire-cli was installed correctly. You can now le
+Note that while any *new* projects you start at this point will use the latest version of Quire just installed, older, previously started projects will remain untouched. This to avoid any unforeseen and unintended changes to publications that are completed or in progress as the version of Quire you use can effect layouts of a page, availablility of features, etcetera.
 
-**Resetting user permission for local directory**
+run commands from the new version of Quire on old Quire projects.
 
-When entering
+While it will not make changes to older projects, you should still be able to run commands from the new version of Quire on old Quire projects. Commands like `quire preview` and `quire pdf`. If you run into trouble with this and find commands do not run as intended or you see errors logged in your command-line, search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) for help.
 
-  ```text
-  sudo chown -R $USER /usr/local
-  ```
+### Update an Existing Project
 
-You will be prompted to enter a password and may receive the following error, *Operation not permitted*.
+If you have an existing project that you want to migrate to a newer version of Quire, we recommend the following process:
 
-Please note, on some computers you may not need to run this command.
+1. Update Quire following the instructions above.
+2. Start a new quire project.
+3. Delete the `content`, `data`, `static` and `layouts` folders in the new project, and replace them with those from your old project.
+4. Copy over any style customization changes you made in the `themes/default/layouts/source/css/variables.scss` file or others. (In older versions of Quire this file would be `themes/quire-starter-theme/layouts/source/css/variables.scss`.)
 
-Try to continue with the installation process to see if everything is running smoothly:
+Run `quire preview --verbose` to test. If you see errors logged in the command-line, or if the preview doesn’t display as intended on http://localhost:1313/, search our [Discussions Forum](https://github.com/thegetty/quire/discussions) for help.
 
-  ```text
-  cd quire-cli
-  npm install -g
-  ```
+## Uninstall Quire
 
-If this works, then you can ignore the *sudo chown* command.
+In your command-line shell (Terminal on macOS and PowerShell in Administrator mode on Windows), type the following text and hit enter:
 
-When you have finished these steps, type the following command to confirm proper installation:
+```text
+npm uninstall --global @thegetty/quire-cli
+```
 
-  ```text
-  quire --version
-  ```
+This will only uninstall Quire itself. External software you installed as part of Quire (inlcuding Node.js, Pandoc, Kindle Previewer, and PrinceXML) will need to be uninstalled individually according to the directions they provide.
```

Comment:
It might be nice to link to those instructions if that is doable, I was trying to uninstall on my personal laptop and got a little lost. Maybe just something to consider as a nice addition in the future and not critical now.